### ISSUE TITLE
[1.17.1] Rendering Crash (no Sodium installed)

### ISSUE THREAD

**Issue by 0gp** [AztechMC/Modern-Industrialization#274]
I have an very annoying bug on my server. Sometimes my client crashes. I hope you can find the solution.
I have removed every Rendering Mod Like Sodium, Indium, Phosphor, Lithium, ...
but the issue still remains.
```
---- Minecraft Crash Report ----
// There are four lights!

Time: 3/24/22, 8:27 AM
Description: Tesselating block in world - Indigo Renderer

java.lang.NullPointerException: Null attachment for pipe rendering! This is not supposed to happen!
Pos: class_2339{x=2996, y=57, z=-6000}
State: Block{modern_industrialization:pipe}
	at aztech.modern_industrialization.pipes.impl.PipeModel.emitBlockQuads(PipeModel.java:102)
	at net.fabricmc.fabric.impl.client.indigo.renderer.render.TerrainRenderContext.tesselateBlock(TerrainRenderContext.java:105)
	at net.minecraft.class_846$class_851$class_4578.redirect$dde000$hookChunkBuildTesselate(class_846.java:723)
	at net.minecraft.class_846$class_851$class_4578.method_22785(class_846.java:524)
	at net.minecraft.class_846$class_851$class_4578.method_22783(class_846.java:447)
	at net.minecraft.class_846$class_851.method_22781(class_846.java:412)
	at net.minecraft.class_846.method_3627(class_846.java:189)
	at net.minecraft.class_761.method_3273(class_761.java:918)
	at net.minecraft.class_761.method_22710(class_761.java:1142)
	at net.minecraft.class_757.method_3188(class_757.java:1026)
	at net.minecraft.class_757.method_3192(class_757.java:810)
	at net.minecraft.class_310.method_1523(class_310.java:1112)
	at net.minecraft.class_310.method_1514(class_310.java:728)
	at net.minecraft.client.main.Main.main(Main.java:217)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:416)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:77)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:210)
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:245)
	at org.multimc.EntryPoint.listen(EntryPoint.java:143)
	at org.multimc.EntryPoint.main(EntryPoint.java:34)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at aztech.modern_industrialization.pipes.impl.PipeModel.emitBlockQuads(PipeModel.java:102)

-- Block being tesselated --
Details:
	Block: Block{modern_industrialization:pipe}
	Block location: World: (2996,57,-6000), Section: (at 4,9,0 in 187,3,-375; chunk contains blocks 2992,0,-6000 to 3007,255,-5985), Region: (5,-12; contains chunks 160,-384 to 191,-353, blocks 2560,0,-6144 to 3071,255,-5633)
Stacktrace:
	at net.fabricmc.fabric.impl.client.indigo.renderer.render.TerrainRenderContext.tesselateBlock(TerrainRenderContext.java:105)
	at net.minecraft.class_846$class_851$class_4578.redirect$dde000$hookChunkBuildTesselate(class_846.java:723)
	at net.minecraft.class_846$class_851$class_4578.method_22785(class_846.java:524)
	at net.minecraft.class_846$class_851$class_4578.method_22783(class_846.java:447)
	at net.minecraft.class_846$class_851.method_22781(class_846.java:412)
	at net.minecraft.class_846.method_3627(class_846.java:189)
	at net.minecraft.class_761.method_3273(class_761.java:918)
	at net.minecraft.class_761.method_22710(class_761.java:1142)
	at net.minecraft.class_757.method_3188(class_757.java:1026)

-- Affected level --
Details:
	All players: 1 total; [class_746['MSandro'/991, l='ClientLevel', x=3006.50, y=60.00, z=-5996.50]]
	Chunk stats: 729, 441
	Level dimension: minecraft:overworld
	Level spawn location: World: (8,81,6), Section: (at 8,1,6 in 0,5,0; chunk contains blocks 0,0,0 to 15,255,15), Region: (0,0; contains chunks 0,0 to 31,31, blocks 0,0,0 to 511,255,511)
	Level time: 1316194 game time, 1316194 day time
	Server brand: fabric
	Server type: Non-integrated multiplayer server
Stacktrace:
	at net.minecraft.class_638.method_8538(class_638.java:370)
	at net.minecraft.class_310.method_1587(class_310.java:2399)
	at net.minecraft.class_310.method_1514(class_310.java:747)
	at net.minecraft.client.main.Main.main(Main.java:217)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:416)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:77)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:210)
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:245)
	at org.multimc.EntryPoint.listen(EntryPoint.java:143)
	at org.multimc.EntryPoint.main(EntryPoint.java:34)

-- Last reload --
Details:
	Reload number: 1
	Reload reason: initial
	Finished: Yes
	Packs: Default, Fabric Mods, KubeJS Resource Pack [assets]

-- System Details --
Details:
	Minecraft Version: 1.17.1
	Minecraft Version ID: 1.17.1
	Operating System: Linux (amd64) version 5.16.14-1-MANJARO
	Java Version: 17.0.3, N/A
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 3592491272 bytes (3426 MiB) / 6970933248 bytes (6648 MiB) up to 8589934592 bytes (8192 MiB)
	CPUs: 16
	Processor Vendor: AuthenticAMD
	Processor Name: AMD Ryzen 7 2700X Eight-Core Processor
	Identifier: AuthenticAMD Family 23 Model 8 Stepping 2
	Microarchitecture: Zen+
	Frequency (GHz): -0.00
	Number of physical packages: 1
	Number of physical CPUs: 8
	Number of logical CPUs: 16
	Graphics card #0 name: Ellesmere [Radeon RX 470/480/570/570X/580/580X/590]
	Graphics card #0 vendor: Advanced Micro Devices, Inc. [AMD/ATI] (0x1002)
	Graphics card #0 VRAM (MB): 258.00
	Graphics card #0 deviceId: 0x67df
	Graphics card #0 versionInfo: unknown
	Virtual memory max (MB): 32273.94
	Virtual memory used (MB): 12957.50
	Swap memory total (MB): 8192.00
	Swap memory used (MB): 0.00
	JVM Flags: 2 total; -Xms512m -Xmx8192m
	Fabric Mods: 
		additionaladditions: Additional Additions 2.2.0
		adorn: Adorn 2.3.1+1.17.1
		advanced_runtime_resource_pack: Runtime Resource Pack 0.5.4
		advancements-enlarger: Advancements Enlarger 0.2.4
		advdebug: Advancements Debug 2.3.0
		ae2wtlib: AE2WTLib 9.0.0-beta.9
		agape_space: Space Dimensions 0.4.2
		alloy_forgery: Alloy Forgery 2.0.6
		appleskin: AppleSkin 2.4.0+mc1.17.1
		appliedenergistics2: Applied Energistics 2 9.0.0-beta.2
		arcanus: Arcanus 1.23
		architectury: Architectury 2.10.7
		artis: Artis-Recrafted 2.1.1+1.17.1
		audiooutput: AudioOutput 0.0.4
		authme: Auth Me 2.1.0
		auto_planter: AutoPlanter 17.1-fabric
		autoconfig1u: Auto Config v1 Updated 3.3.1
		awesomedungeon: Awesome Dungeon 2.0.5
		balm-fabric: Balm 1.9.0+0
		bannerpp: Banner++ 2.0.4+mc.1.17-rc1
		bcf: Balanced Cobblestone Funnels Release 1 Fix 3
		bclib: BCLib 0.5.6
		beaconoverhaul: Beacon Overhaul 1.4.3+1.17
		better_wandering_trader: Better Wandering Trader 1.7.0
		betteranimalsplus: Better Animals Plus 1.17.1-11.0.5
		betterdungeons: YUNG's Better Dungeons 1.17-1.0.2
		betterend: Better End 0.12.5
		bettermineshafts: YUNG's Better Mineshafts 1.17-1.0.1
		betternether: Better Nether 5.3.14
		bettersleeping: Better Sleeping 0.4.0
		bewitchment: Bewitchment 1.17-15
		bitsandchisels: BitsAndChisels 2.6.0
		blockshifter: Blockshifter 1.4.0+fabric-1.17.1
		blockus: Blockus 2.2.10+1.17.1
		blue_endless_jankson: jankson 1.2.1
		botania: Botania 1.17.1-422-FABRIC
		bountiful: Bountiful 2.0.2
		breakprogress: Break Progress 1.0.1
		buildinggadgets: st'ructure tools 1.0.5
		builtinservers: Builtin Servers 1.0.1-BETA+1.17
		campanion: Campanion 2.0.1
		can-i-mine-this-block: cAn i MiNe thIS bLOCk? 1.1.0
		cardinal-components-base: Cardinal Components API (base) 3.1.1
		cardinal-components-block: Cardinal Components API (blocks) 3.0.0
		cardinal-components-entity: Cardinal Components API (entities) 3.1.1
		cardinal-components-item: Cardinal Components API (items) 3.1.1
		cardinal-components-level: Cardinal Components API (world saves) 3.1.1
		cardinal-components-world: Cardinal Components API (worlds) 3.1.1
		carrier: Carrier 1.8.2
		charm: Charm 3.3.2
		chisel: Chisel Refabricated 1.4.5
		cloth-api: Cloth API 2.0.54
		cloth-armor-api-v1: Cloth Armor API v1 1.5.47
		cloth-basic-math: cloth-basic-math 0.6.0
		cloth-client-events-v0: Cloth Client Events v0 2.0.54
		cloth-common-events-v1: Cloth Common Events v1 2.0.54
		cloth-config: Cloth Config v6 6.1.48
		cloth-datagen-api-v1: Cloth Datagen v1 2.0.54
		cloth-scissors-api-v1: Cloth Scissors API v1 2.0.54
		cloth-utils-v1: Cloth Utils v1 2.0.54
		collective-fabric: Collective (Fabric) 4.26
		com_electronwill_night-config_core: core 3.6.5
		com_electronwill_night-config_toml: toml 3.6.5
		com_eliotlash_mclib_mclib: mclib 18
		com_eliotlash_molang_molang: molang 18
		com_focamacho_sealconfig: sealconfig 1.2.2
		com_moandjiezana_toml_toml4j: toml4j 0.7.2
		com_typesafe_config: config 1.4.1
		completeconfig: CompleteConfig 1.2.0
		compostable-rottenflesh: Compostable Rotten Flesh 1.0.0
		compress: Compress 'em [v1.5]
		computercraft: CC:Restitched 1.97.5
		confabricate: confabricate 2.2.0-SNAPSHOT+4.1.1
		conjuring: Conjuring 1.0.16
		couplings: Couplings 1.6.0+1.17
		crookedcrooks: Crooked Crooks 2.0.1+1.17.1
		croparia: Croapria 1.17.1-0.0.4
		croptopia: Croptopia 1.7.0
		crowdin-translate: CrowdinTranslate 1.3+1.17
		customportalapi: Custom Portal Api 0.0.1-beta46-1.17
		customspawns: Custom Spawns 0.5.1+1.17
		danceofgrowth: Twerk / Crop Dusting Mod 1.0
		dark-enchanting: Dark Enchanting 0.6.0-1.17.1
		datapackportals: Datapack Portals 1.1.0-1.17
		dazzle: Dazzle 2 1.0
		diggusmaximus: Diggus Maximus 1.5.1-1.17
		dispenser_configurator: Dispenser Configurator 1.2-b1+MC1.17-1.17.1
		dml-refabricated: Deep Mob Learning: Refabricated 0.4.4-beta
		dml-refabricated-base: DML: Refabricated - Base 0.4.4-beta
		dml-refabricated-events: DML: Refabricated - Vanilla Events 0.4.4-beta
		dml-refabricated-modular-armor: DML: Refabricated - Glitch Armor 0.4.4-beta
		dml-refabricated-rei: DML: Refabricated - REI Integration 0.4.4-beta
		drippyloadingscreen: Drippy Loading Screen 1.4.1
		endrem: End Remastered 4.1.0
		enhanced_quarries: Enhanced Quarries 1.2.0_RC9
		expandedstorage: Expanded Storage 7.3.6
		extraalchemy: Extra Alchemy 1.6.1
		extragenerators: Extra Generators 1.1-BETA+1.17
		extremesoundmuffler: Extreme sound muffler 3.22_fabric-1.17.1
		fabric: Fabric API 0.46.1+1.17
		fabric-api-base: Fabric API Base 0.4.0+cf39a74318
		fabric-api-lookup-api-v1: Fabric API Lookup API (v1) 1.5.0+e821752d18
		fabric-biome-api-v1: Fabric Biome API (v1) 3.2.2+cf39a74318
		fabric-blockrenderlayer-v1: Fabric BlockRenderLayer Registration (v1) 1.1.6+cf39a74318
		fabric-command-api-v1: Fabric Command API (v1) 1.1.4+cf39a74318
		fabric-commands-v0: Fabric Commands (v0) 0.2.3+cf39a74318
		fabric-containers-v0: Fabric Containers (v0) 0.1.13+cf39a74318
		fabric-content-registries-v0: Fabric Content Registries (v0) 0.4.1+3447790d18
		fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.1.6+cf39a74318
		fabric-dimensions-v1: Fabric Dimensions API (v1) 2.0.15+b556f28c18
		fabric-entity-events-v1: Fabric Entity Events (v1) 1.4.1+377137cc18
		fabric-events-interaction-v0: Fabric Events Interaction (v0) 0.4.12+e99fbe1218
		fabric-events-lifecycle-v0: Fabric Events Lifecycle (v0) 0.2.4+cf39a74318
		fabric-furnaces: Fabric Furnaces 2.0.1-1.17
		fabric-game-rule-api-v1: Fabric Game Rule API (v1) 1.0.8+cf39a74318
		fabric-item-api-v1: Fabric Item API (v1) 1.3.0+6617390918
		fabric-item-groups-v0: Fabric Item Groups (v0) 0.3.2+cf39a74318
		fabric-key-binding-api-v1: Fabric Key Binding API (v1) 1.0.6+2a2bb57318
		fabric-keybindings-v0: Fabric Key Bindings (v0) 0.2.4+cf39a74318
		fabric-language-kotlin: Fabric Language Kotlin 1.7.1+kotlin.1.6.10
		fabric-language-scala: Fabric Language Scala 2.0.0+scala.3.0.2
		fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 1.4.6+0392f3a618
		fabric-loot-tables-v1: Fabric Loot Tables (v1) 1.0.5+cf39a74318
		fabric-mining-level-api-v1: Fabric Mining Level API (v1) 1.0.3+cf39a74318
		fabric-mining-levels-v0: Fabric Mining Levels (v0) 0.1.7+cf39a74318
		fabric-models-v0: Fabric Models (v0) 0.3.1+cf39a74318
		fabric-networking-api-v1: Fabric Networking API (v1) 1.0.14+5847535018
		fabric-networking-blockentity-v0: Fabric Networking Block Entity (v0) 0.2.12+cf39a74318
		fabric-networking-v0: Fabric Networking (v0) 0.3.3+cf39a74318
		fabric-object-builder-api-v1: Fabric Object Builder API (v1) 1.11.1+f907116918
		fabric-object-builders-v0: Fabric Object Builders (v0) 0.7.8+cf39a74318
		fabric-particles-v1: Fabric Particles (v1) 0.2.5+cf39a74318
		fabric-permissions-api-v0: fabric-permissions-api 0.1-SNAPSHOT
		fabric-registry-sync-v0: Fabric Registry Sync (v0) 0.8.0+ea29b33318
		fabric-renderer-api-v1: Fabric Renderer API (v1) 0.4.6+ce16ae5918
		fabric-renderer-indigo: Fabric Renderer - Indigo 0.4.10+ce16ae5918
		fabric-renderer-registries-v1: Fabric Renderer Registries (v1) 3.2.6+cf39a74318
		fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.1.6+cf39a74318
		fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 0.2.1+cf39a74318
		fabric-rendering-v0: Fabric Rendering (v0) 1.1.7+cf39a74318
		fabric-rendering-v1: Fabric Rendering (v1) 1.10.1+377137cc18
		fabric-resource-conditions-api-v1: Fabric Resource Conditions API (v1) 1.0.1+591a0f2118
		fabric-resource-loader-v0: Fabric Resource Loader (v0) 0.4.10+f09604ce18
		fabric-screen-api-v1: Fabric Screen API (v1) 1.0.5+cf39a74318
		fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.1.9+cf39a74318
		fabric-structure-api-v1: Fabric Structure API (v1) 1.2.0+48a9ad8918
		fabric-tag-extensions-v0: Fabric Tag Extensions (v0) 1.2.3+cf39a74318
		fabric-textures-v0: Fabric Textures (v0) 1.0.7+cf39a74318
		fabric-tool-attribute-api-v1: Fabric Tool Attribute API (v1) 1.3.3+3b96517518
		fabric-transfer-api-v1: Fabric Transfer API (v1) 1.5.4+cf39a74318
		fabrication: Fabrication 2.3.0
		fabricloader: Fabric Loader 0.13.3
		factory: Factory 1.17.1-1.4.0
		fake-player-api: Fake Player API 0.2.0
		farmersdelight: Farmer's Delight 1.17.1-0.1.3
		fasttransferlib: Fast Transfer Lib 0.5.6
		fiber: fiber 0.23.0-2
		fiblib: FibLib 1.2.1+1.17
		game-phases: Game Phases 1.1.0-1.17.1
		gargantuan: Gargantuan 1.1.4
		gateofbabylon: Gate Of Babylon 1.5.1-1.17.1
		gearreborn: Gear Reborn 1.17-0.1.10
		geckolib3: Geckolib 3.0.32
		glassential: Glassential 1.17-1.2.0
		go-fish: Go Fish 1.3.1-1.17.1
		golemsgalore: Golems Galore 1.0.3-1.17.1
		grass_pass: Grass Bypass 1.0.0
		grassseeds-fabric: Grass Seeds (Fabric) 1.6
		gud_recipe_confidence: Recipe Confidence 1.0.2
		guild: Guild 0.3.4
		hardcorequesting: Hardcore Questing Mode 1.17.1-5.8.2
		hctm-base: HCTM Base 3.6.2
		hologram-api: Hologram API 0.2.1+1.17.1
		iknowwhatimdoing: I Know What I'm Doing 4.0.0
		impersonate: Impersonate 2.3.5
		improved-stations: Improved Stations 2.0.7
		incubus_core: Incubus Core 1.6.3
		indrev: Industrial Revolution 1.12.0-BETA
		inmis: Inmis 2.3.2-1.17.1
		inventorysorter: Inventory Sorter 1.8.0-1.17
		io_leangen_geantyref_geantyref: geantyref 1.3.11
		iron-jetpacks: Iron Jetpacks 0.4.2
		jankson: Jankson 3.0.1+j1.2.0
		java: OpenJDK 64-Bit Server VM 17
		javax_servlet_javax_servlet-api: javax.servlet-api 3.1.0
		justenoughkeys: Just Enough Keys 0.5.3
		kambrik: Kambrik 2.4.0
		kanos_config: Kanos Config 0.1.4+1.14.4-1.17.1
		kibe: Kibe 1.9.4-BETA+1.17
		kjsextras: KubeJS Extras 1.4.0
		konkrete: Konkrete 1.3.0
		ktdatataglib: KtDataTagLib 1.5.4
		kubejs: KubeJS 1701.3.10-build.9999
		kyrptconfig: Kytpt Config 1.1.10-1.17
		lapisreserve: Lapis Reserve 1.0.8
		libaddict: libaddict 0.3.0
		libblockattributes: LibBlockAttributes 0.9.0
		libblockattributes_core: LibBlockAttributes (Core) 0.9.0
		libblockattributes_fluids: LibBlockAttributes (Fluids) 0.9.0
		libblockattributes_items: LibBlockAttributes (Items) 0.9.0
		libcd: LibCapableData 3.0.3+1.16.3
		libgui: LibGui 4.2.3+1.17.1
		libmultipart: LibMultiPart 0.6.0
		libnetworkstack: Lib Network Stack 0.5.0
		libninepatch: LibNinePatch 1.1.0
		libraryferret: Library Ferret 1.0.3
		lifts: Lifts 1.4-BETA+1.17
		lightoverlay: Light Overlay 6.0.1
		locki: Locki 0.5.0
		magna: Magna 1.5.5-1.17-pre4
		mambience: MAmbience 3.2.0+1.17.1
		mcdict: MCDict 2.1.0+1.17.1
		mechanicaldispensers: MechanicalDispensers 3.6
		megane: megane 5.6.1
		megane-applied-energistics-2: megane-applied-energistics-2 5.6.1+9.0.0-beta.1
		megane-base: megane-base 5.6.1
		megane-extra-generators: megane-extra-generators 5.6.1+1.1-BETA-1.17
		megane-fabric-transfer: megane-fabric-transfer 5.6.1+1.5.1-6d0fac4218
		megane-fast-transfer-lib: megane-fast-transfer-lib 5.6.1+0.5.7
		megane-industrial-revolution: megane-industrial-revolution 5.6.1+1.11.0-alpha.4
		megane-kibe: megane-kibe 5.6.1+1.9.3-BETA-1.17
		megane-lib-block-attributes: megane-lib-block-attributes 5.6.1+0.9.0
		megane-modern-industrialization: megane-modern-industrialization 5.6.1+0.6.1
		megane-reborn-core: megane-reborn-core 5.6.1+5.0.11-beta
		megane-runtime: megane-runtime 5.6.1
		megane-team-reborn-energy: megane-team-reborn-energy 5.6.1+2.0.0-beta1
		megane-tech-reborn: megane-tech-reborn 5.6.1+5.0.11-beta
		megane-vanilla: megane-vanilla 5.6.1+1.17.1
		megane-wireless-networks: megane-wireless-networks 5.6.1+1.3.1
		minecraft: Minecraft 1.17.1
		minecraftmappings: Minecraft Mappings 1.17.1-1.0.0
		minimalmenu: MinimalMenu 1.17-0.0.8
		mm: Manningham Mills 2.1
		mo_glass: Mo Glass 1.6.1-MC1.17.1
		modern_industrialization: Modern Industrialization 0.7.4
		modmenu: Mod Menu 2.0.14
		more_tooltips: MoreTooltips 1.0.2
		moretags: More Tags 1.2.5
		morevillagers-fabric: MoreVillagersFabric 2.1.1-SNAPSHOT
		morevillagesmod: More Villages Mod 1.0.0
		mostructures: Mo' Structures 1.3.0-pre1-1.17.1
		mousetweaks: Mouse Tweaks 2.15
		mtr: Minecraft Transit Railway 1.17.1-3.0.0-beta-8
		myron: Myron 1.6.1
		nametagtweaks-fabric: Name Tag Tweaks (Fabric) 1.7
		nbtcrafting: NBT Crafting 2.1.3+mc1.17.1
		netherportalfix: NetherPortalFix 8.0.0+0
		ninjaphenix_container_lib: NinjaPhenix's Container Library 1.2.10
		observable: Observable 0.1.2-1.17
		offhandoverhaul: Offhand Overhaul 1.2.0
		oldfishfarm: One Block Fish Farm 1.0.0
		omega-config: OmegaConfig 1.0.8
		org_codehaus_groovy_groovy: groovy 3.0.8
		org_codehaus_groovy_groovy-jsr223: groovy-jsr223 3.0.8
		org_eclipse_jetty_jetty-http: jetty-http 9.4.44.v20210927
		org_eclipse_jetty_jetty-io: jetty-io 9.4.44.v20210927
		org_eclipse_jetty_jetty-security: jetty-security 9.4.44.v20210927
		org_eclipse_jetty_jetty-server: jetty-server 9.4.44.v20210927
		org_eclipse_jetty_jetty-servlet: jetty-servlet 9.4.44.v20210927
		org_eclipse_jetty_jetty-util: jetty-util 9.4.44.v20210927
		org_eclipse_jetty_jetty-util-ajax: jetty-util-ajax 9.4.44.v20210927
		org_jetbrains_annotations: annotations 19.0.0
		org_jetbrains_kotlin_kotlin-reflect: kotlin-reflect 1.6.10
		org_jetbrains_kotlin_kotlin-stdlib: kotlin-stdlib 1.6.10
		org_jetbrains_kotlin_kotlin-stdlib-jdk7: kotlin-stdlib-jdk7 1.6.10
		org_jetbrains_kotlin_kotlin-stdlib-jdk8: kotlin-stdlib-jdk8 1.6.10
		org_jetbrains_kotlinx_kotlinx-coroutines-core-jvm: kotlinx-coroutines-core-jvm 1.5.2
		org_jetbrains_kotlinx_kotlinx-coroutines-jdk8: kotlinx-coroutines-jdk8 1.5.2
		org_jetbrains_kotlinx_kotlinx-serialization-cbor-jvm: kotlinx-serialization-cbor-jvm 1.3.1
		org_jetbrains_kotlinx_kotlinx-serialization-core-jvm: kotlinx-serialization-core-jvm 1.3.1
		org_jetbrains_kotlinx_kotlinx-serialization-json-jvm: kotlinx-serialization-json-jvm 1.3.1
		org_scala-lang_scala-library: scala-library 2.13.6
		org_scala-lang_scala3-library_3: scala3-library_3 3.0.2
		org_spongepowered_configurate-core: configurate-core 4.1.1
		org_spongepowered_configurate-extra-dfu4: configurate-extra-dfu4 4.1.1
		org_spongepowered_configurate-gson: configurate-gson 4.1.1
		org_spongepowered_configurate-hocon: configurate-hocon 4.1.1
		outvoted: Outvoted 1.5.0-rc.1
		owo: oωo 0.3.2+1.17
		oxidizelib: OxidizeLib 1.0.0
		packet_tweaker: Packet Tweaker 0.2.0+1.17.1
		patchouli: Patchouli 1.17.1-59-FABRIC
		pehkui: Pehkui 2.6.0+1.14.4-1.18
		placeholder-api: Placeholder API 1.1.2+1.17.1
		platforms: Platforms 1.1.2
		playerabilitylib: Pal 1.4.0
		polymer: Polymer 0.1.8+1.17.1
		pride: Pride Lib 1.0.0+21w03a
		pugh_lib: Pugh Lib ${version}
		qcommon-cfg-core: qcommon-cfg-core 3.3.0
		qcommon-croco: Croco 2.1.1
		randombonemealflowers-fabric: Random Bone Meal Flowers (Fabric) 1.7
		reach-entity-attributes: Reach Entity Attributes 2.1.1
		reborncore: Reborn Core 5.0.11-beta
		recast-fabric: Recast (Fabric) 1.5
		redbits: Red Bits 1.10.1
		redstonebits: Redstone Bits 1.5.0
		repurposed_structures: Repurposed Structures 2.5.7+1.17.1
		resourceful_tools: Resourceful Tools 1.1.23
		rhino: Rhino 1701.1.5-build.71
		right-click-harvest: Right-Click-Harvest 1.1.0
		ringsofascension: Rings of Ascension 1.0.1
		roughlyenoughitems: Roughly Enough Items 6.4.426
		rswires: RSWires 4.11.2
		scf: Super Cobblestone Funnels 2.1.0
		searchlight: Searchlight 1.1.3
		seasons: Fabric Seasons 1.3.1-BETA+1.17
		seedy-behavior: Seedy Behavior 1.0.1
		server_translations_api: Server Translations API 1.4.8+1.17
		sgui: sgui 1.0.0-rc5+1.17.1
		shetiphiancore: ShetiPhianCore 1.1.3
		shutupconsole: Shut Up Console 1.1
		simple_pipes: SimplePipeTest 0.6.2
		simple_pipes_dep_container: SimplePipeTest (Dependency Container) 0.6.2
		simpleteleporters: Simple Teleporters 2.3.0
		skyutils: Sky utilities Mod 1.0.5_beta4
		skyvillages: Sky Villages 1.0.1.1-fabric
		slight-gui-modifications: 'Slight' GUI Modifications 2.0.3
		smart-recipes: Smart Recipes 0.1.1+1.17
		someforgepatchesported: Some Forge Patches Ported 1.0.1
		somnus: Somnus API 0.0.14-1.17.1
		spatialharvesters: Spatial Harvesters 0.11.12d
		stacc: Stacc 1.2.3
		starry_sky: Starry Skies 1.3.0-1.17.1
		steedstopper: Steed Stopper 1.0.1
		step-height-entity-attribute: Step Height Entity Attribute 1.0.0
		stoneholm: Stoneholm 1.3
		stonevaults: Stonevaults 1.1.0
		storagecabinet: Storage Cabinet 1.4.2
		structureworld: Structure World 1.1.1-BETA+1.17
		structurized: Structurized 1.3.0
		sync: Sync (Fabric) 3.1
		team_reborn_energy: Energy 2.1.0
		techreborn: Tech Reborn 5.0.11-beta
		terraform-wood-api-v1: Terraform Wood API (v1) 2.0.2
		terrarianslimes: Terrarian Slimes 1.1-BETA+1.17
		the_aether: Paradise Lost - The Aether Reborn 1.6.5.3+1.17.1
		the_bumblezone: The Bumblezone - Fabric 3.4.1+1.17.1
		things: Things 0.2.7
		timeinabottle: Time in a Bottle 1.6-mc1.17-fabric
		toms_storage: Tom's Simple Storage Mod 1.1.18
		totemmod: Wither Totems Mod 0.3.2
		trade: Trade 1.2.1
		trapexpansion: Trap Expansion 1.3.3
		trinkets: Trinkets 3.0.4
		truerooms: True Secret Rooms 1.1.5
		universal-graves: Universal Graves 1.1.4+1.17.1
		unlimiteddragoneggs: Unlimited Dragon Eggs 1.2.1
		valley: ValleyCraft 2.0.3
		victus: Victus 0.1.4+1.17.1
		wands: Building Wands 2.2.8_mc1.17.1
		waystones: Waystones 2.2.3
		wireless_redstone: Wireless Redstone 1.0.1
		wirelessnetworks: Wireless Networks 1.3.1
		wmitaf: WMITAF 1.2.0
		workings: Workings 1.0.7
		wthit: wthit 3.11.2
		xaerominimap: Xaero's Minimap 22.3.1.1
		xaeroworldmap: Xaero's World Map 1.20.4.1
		yosbr: YOSBR 0.1.1
		yungsapi: YUNG's API 1.17-Fabric-19
		zombievillagersfromspawner-fabric: Zombie Villagers From Spawner (Fabric) 1.6
	Launched Version: MultiMC5
	Backend library: LWJGL version 3.2.2 build 10
	Backend API: AMD Radeon RX 580 Series (POLARIS10, DRM 3.44.0, 5.16.14-1-MANJARO, LLVM 13.0.1) GL version 4.6 (Core Profile) Mesa 21.3.7, AMD
	Window size: 850x640
	GL Caps: Using framebuffer using OpenGL 3.2
	GL debug messages: 
	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'fabric'
	Type: Client (map_client.txt)
	Graphics mode: fancy
	Resource Packs: vanilla, Fabric Mods, kubejs:resource_pack (incompatible)
	Current Language: English (US)
	CPU: 16x AMD Ryzen 7 2700X Eight-Core Processor
```

**Comment by 0gp** [comment]
#150

**Comment by Technici4n** [comment]
Should be fixed by 0.7.5 (will be out in a few minutes).
